noir/noir-repo/test_programs/execution_success/brillig_acir_loop/src/main.nr

```diff
@@ -0,0 +1,14 @@
+fn main(x: Field, y: pub Field) {
+    for _ in 0..4 {
+        foo(x, y);
+        check_values(x, y);
+    }
+}
```

Comment:
What does this test check for? That the loop doesn't generate extra copies of the brillig byte code?

If so I wonder if this could be a unit test somehow so we could attempt to check for that directly since this only tests that it executes correctly.

Reply:
Yeah I was using it to test my codegen, but you are correct that in `execution_success` it doesn't make much sense so I am going to move it to a unit test in ACIR gen.

Reply:
Made a unit test and deleted this test as it is unnecessary